File: Plus_Minus/Plus_Minus.py
#!/bin/python3

import math
import os
import random
import re
import sys

#
# Complete the 'plusMinus' function below.
#
# The function accepts INTEGER_ARRAY arr as parameter.
#

def plusMinus(arr):
    # Write your code here
    gt = 0
    et = 0
    lt = 0
    count = 0
    for a in arr:
        count += 1
        if a > 0:
            gt += 1;
        elif a < 0:
            lt += 1;
        else:
            et += 1;
    answer = gt / count;
    answer2 = lt / count;
    answer3 = et / count;
    # lt / count, 
    # et / count;
    print(round(answer, 6))
    print(round(answer2, 6))
    print(round(answer3, 6))


if __name__ == '__main__':
    n = int(input().strip())

    arr = list(map(int, input().rstrip().split()))

    plusMinus(arr)

File: JavaScript/Plus_Minus/Plus_Minus.js
'use strict';

process.stdin.resume();
process.stdin.setEncoding('utf-8');

let inputString = '';
let currentLine = 0;

process.stdin.on('data', function(inputStdin) {
    inputString += inputStdin;
});

process.stdin.on('end', function() {
    inputString = inputString.split('\n');

    main();
});

function readLine() {
    return inputString[currentLine++];
}

/*
 * Complete the 'plusMinus' function below.
 *
 * The function accepts INTEGER_ARRAY arr as parameter.
 */

function plusMinus(arr) {
    // Write your code here
    let gt = 0
    let et = 0
    let lt = 0
    let newArr = arr.map((a) => {
        if (a > 0) {
            gt++
        } else if (
            a < 0
        ) {
            lt++
        } else {
            et++
        }
    })
    console.log((gt/arr.length).toFixed(6)) 
    console.log((lt/arr.length).toFixed(6))
    console.log((et/arr.length).toFixed(6))
}

function main() {
    const n = parseInt(readLine().trim(), 10);

    const arr = readLine().replace(/\s+$/g, '').split(' ').map(arrTemp => parseInt(arrTemp, 10));

    plusMinus(arr);
}